test/adaptive_load/utility.cc

```diff
@@ -0,0 +1,42 @@
+#include "test/adaptive_load/utility.h"
+
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Nighthawk {
+
+namespace {
+
+void SetCounterValue(nighthawk::client::Counter* counter, absl::string_view name, int value) {
+  counter->set_name(std::string(name));
+  counter->set_value(value);
+}
+
+void SetStatisticValues(nighthawk::client::Statistic* statistic, absl::string_view id, long min_ns,
+                        long mean_ns, long max_ns, long pstdev_ns) {
+  statistic->set_id(std::string(id));
+  *statistic->mutable_min() = Envoy::Protobuf::util::TimeUtil::NanosecondsToDuration(min_ns);
+  *statistic->mutable_mean() = Envoy::Protobuf::util::TimeUtil::NanosecondsToDuration(mean_ns);
+  *statistic->mutable_max() = Envoy::Protobuf::util::TimeUtil::NanosecondsToDuration(max_ns);
+  *statistic->mutable_pstdev() = Envoy::Protobuf::util::TimeUtil::NanosecondsToDuration(pstdev_ns);
+}
+
+} // namespace
+
+nighthawk::client::Output MakeSimpleNighthawkOutput(const SimpleNighthawkOutputSpec& spec) {
```

Comment:
Might be helpful to have some comment on this one. I didn't find this as clear as the other two.

Reply:
This one is commented over in utility.h.